pkg/rc/farm.go

```diff
@@ -200,6 +202,31 @@ START_LOOP:
 	}
 }
 
+func (rcf *Farm) failsafe(rcFields []fields.RC) {
+	// FAILSAFES. If no RCs are scheduled, or there are zero replicas of anything scheduled, panic
+	if len(rcFields) == 0 {
+		rcf.alerter.Alert(alerting.AlertInfo{
+			Description: "No RCs have been scheduled",
+			IncidentKey: "no_rcs_found",
+		})
+		panic("No RCs are scheduled at all. Create one RC to enable the farm. Panicking to escape a potentially bad situation.")
+	}
+	globalReplicaCount := 0
+	for _, rc := range rcFields {
+		if rc.ReplicasDesired > 0 {
+			return
+		}
+		globalReplicaCount += rc.ReplicasDesired
+	}
+	if globalReplicaCount == 0 {
```

Comment:
I guess that makes alert message more difficult and it's an unlikely scenario, so i'm fine with you keeping it